pkg/config/gui-server/config.go

```diff
@@ -10,7 +10,7 @@ type GuiServerConfig struct {
 	// Port on which the server is exposed
 	Port uint32 `yaml:"port" envconfig:"kuma_gui_server_port"`
 	// Config of the GUI itself
-	GuiConfig *GuiConfig
+	GuiConfig *GuiConfig `yaml:"-"`
```

Comment:
Otherwise the json/yaml looks wierd. Like this:
```
{
  "guiServer": {
      "port": 1234,
      "GuiConfig": {
          "ApiUrl": "",
          "Environment": "",
      }
  }
}
```
everything in config is lowercase instead of this. If we write `yaml` tags, we will expose this config to the users.

Reply:
Up to you.
Personally, I see the value in exposing the entire config even if it comes at a cost.